### ISSUE TITLE
light.el error on read-file-name-internal

### ISSUE THREAD

**Issue by jdchristensen** [jhpalmieri/ultratex#4]
In `light.el`, the following appears twice:  `(read-file-name-internal str dir action)`

Here, `dir` is a string.  I get an error:
```
completion--some: Invalid function: "/path/to/current/directory/"
```

But maybe this is something special about my set-up.  (E.g. because I use Ido completion as well?)

**Comment by jhpalmieri** [comment]
I don't see this. When do you get the error? Just upon loading the file, or when completing on file names, or some other time?

**Comment by jdchristensen** [comment]
If I run `emacs -q` to ignore my init file, and evaluate the following
```
(add-to-list 'load-path "~/emacs/ultratex/lisp")
(require 'light)
```
then if I put point after `/tm` in a buffer and run `M-x completing-read-file-name` I get
```
Making completion list...
completion--some: Invalid function: "/home/jdc"
```
where the directory shown is my current folder.  That's the `dir` argument, which it seems is expected to be a predicate.

Long ago I replaced `dir` with `nil` and things have worked fine for me, but I don't know what is intended there.

**Comment by jdchristensen** [comment]
This also happens with Emacs 27 and probably older emacsen.

**Comment by jdchristensen** [comment]
Sorry, that should have been `completing-insert-file-name`.

**Comment by jhpalmieri** [comment]
I think the problem is that `read-file-name-internal` is, as the name implies, internal, with documentation saying "do not call this." So it's not well documented and it's also subject to change. In fact I think that it changed at some point, and that's what is causing the problems.

Your solution of replacing `dir` by `nil` seems to work, so let's do that.

**Comment by jhpalmieri** [comment]
Fixed by pr #14

**Comment by jdchristensen** [comment]
Thanks!
